yaml/com/ibm/VPD/FRUManager.interface.yaml

```diff
@@ -0,0 +1,129 @@
+description: >
+    Implement to manage VPD in the system.
+
+methods:
+    - name: WriteKeyword
+      description: >
+          Method to update the keyword value with the given value. Updates the
+          keyword on hardware and also updates on the cache, if cache
+          information is present in the config JSON.
+      parameters:
+          - name: path
+            type: string
+            description: >
+                Inventory/hardware path of the FRU whose keyword needs to be
+                updated.
+          - name: data
+            type:
+                variant[struct[string,string,array[byte]],
+                struct[string,array[byte]]]
+            description: >
+                Contains data required to write. currently supports IPZ and
+                Keyword types VPD and more can be added.
+      errors:
+          - xyz.openbmc_project.Common.Error.InvalidArgument
+          - com.ibm.VPD.Error.EccCheckFailed
+
+    - name: ReadKeyword
+      description: >
+          Method to read VPD keyword value. Keyword can be read from hardware or
+          from cache.
+      parameters:
+          - name: path
+            type: string
+            description: >
+                Inventory/hardware Path of the FRU whose keyword needs to be
+                read.
+          - name: data
+            type: variant[struct[string,string],struct[string]]
+            description: >
+                Contains data required to read. currently supports IPZ and
```

Comment:
"Information required to read keyword value."
I feel this is good .

Reply:
updated